src/main/java/hashtable/HashTable.java

```diff
@@ -41,18 +48,21 @@ public void put(K key, V value) {
             node.next.prev = node;
         }
         array.set(index, node);
+        count++;
+        double loadFactor = (1.0 * count) / initialCapacity;
```

Comment:
`(1.0 * count)` did you mean to do `((double) count)`? Also why do you need additional field `initialCapacity`? Why not use `array.length`? Seems like there's something logically wrong with these lines..